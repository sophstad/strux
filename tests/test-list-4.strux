int main() {
	LinkedList::string:: l = new LinkedList::string::();
	l.add("HAPPY");
	string s = l.get(0);
	print(s);
	return 0;
}