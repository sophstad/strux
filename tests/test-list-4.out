HAPPY
[ S ] -> [ T ] -> [ R ] -> [ U ] -> [ X ]
5
